packages/ui5-application-writer/src/data/index.ts

```diff
@@ -18,6 +18,10 @@ export function mergeWithDefaults(ui5App: Ui5App): {
     validate(ui5App);
     ui5App.app = mergeApp(ui5App.app);
     ui5App.appOptions = ui5App.appOptions || {};
+    // if typescript and codeAssist is enabled disable codeAssist
+    if (ui5App.appOptions.typescript && ui5App.appOptions.codeAssist) {
```

Comment:
Should we document this behaviour via the types ts docs: 
https://github.com/SAP/open-ux-tools/blob/main/packages/ui5-application-writer/src/types.ts#L57

Reply:
What if I set `codeAssist` to false and 'typescript' to true? I assume we still get code assist...

Reply:
Yes that would be the same result as both being true. You get the Typescript code assist enabled.

Reply:
Updated in https://github.com/SAP/open-ux-tools/pull/773/commits/daff85d11c822dec010b2b6ef3f006356d0472c4